docs/fr/users/01_Installation.md

```diff
@@ -119,7 +119,9 @@ server {
 		fastcgi_split_path_info ^(.+\.php)(/.*)$;
 		# Par défaut la variable PATH_INFO n’est pas définie sous PHP-FPM
 		# or l’API FreshRSS greader.php en a besoin. Si vous avez un “Bad Request”, vérifiez bien cette dernière !
-		fastcgi_param PATH_INFO $fastcgi_path_info;
+		# REMARQUE: l'utilisation de la variable $path_info distincte est nécessaire
```

Comment:
In French, there is a space before a colon!

Reply:
Apologies, I'm relying on Google Translate for the French config
Feel free to make any adjustments ;)